### PR TITLE
Haafingar

### PR DESCRIPTION
**Pull request by Kurash1**

### CODE REVIEW

File: src/content/nords/haafingar/conquest_missions.txt [hunk 1]
```diff
@@ -356,6 +199,29 @@ The blessing of the steed is a blessing of war, and thus it is only right for th
...
+		trigger = {
+			NOT = {
+				exists = lone_band
+			}
+			provinces:lone_isle = {
+				country_or_non_sovereign_subject_holds = ROOT
+			}
+		}
+		effect = {
+
```
Comment (JoMakka) [inline src/content/nords/haafingar/conquest_missions.txt:222]: no effect?

Comment (Kurash1) [reply src/content/nords/haafingar/conquest_missions.txt:222]: none yet, the mission tree is still WIP

File: src/interface/countrytechnologyview.txt [hunk 2]
```diff
@@ -708,6 +708,63 @@ new interface_file countrytechnologyview = {
 			scripted = yes
 		}
 		
+		instantTextBoxType = {
+			name = "survey_idea_army_text"
+			position = { x = (300 + 128 - 118) y = (128 + 148 - 155 - 6) }
```
Comment (JoMakka) [inline src/interface/countrytechnologyview.txt:713]: what? U didn't want to do math?

Comment (Kurash1) [reply src/interface/countrytechnologyview.txt:713]: I mean who would, just let arc handle it.

Comment (JoMakka) [reply src/interface/countrytechnologyview.txt:713]: well it looks less elegant ,wouldn't u agree?

Comment (Kurash1) [reply src/interface/countrytechnologyview.txt:713]: I mean kinda, but velocity is more important in this situation, as the position of elements can keep getting adjusted.

Comment (JoMakka) [reply src/interface/countrytechnologyview.txt:713]: Fair enough

File: src/systems/adm tech.txt [hunk 3]
```diff
@@ -13,28 +13,87 @@ new custom_transpile adm_technology = {
 			may_force_march = yes
 			allowed_idea_groups = 1
 			
-			foreach $building in buildings {
```
Comment (JoMakka) [inline src/systems/adm tech.txt:16]: no more foreach

Comment (JoMakka) [reply src/systems/adm tech.txt:16]: sadge

**Review by JoMakka — APPROVED** [review]
LGTM

**Pull request merged**